docs/UserGuide.md

```diff
@@ -574,7 +574,8 @@ Format: `sort TYPE [ORDER]`
   - `DESC`
   - If it is left as blank, it will follow the default order based on the specified`TYPE`.
```

Comment:
Seems to have a missing space in this line? specifiedTYPE

Reply:
Noted! Good catch haha